core/controllers/suggestion.py

```diff
@@ -84,14 +89,41 @@ class SuggestionHandler(base.BaseHandler):
     @acl_decorators.can_suggest_changes
     def post(self):
         try:
-            suggestion_services.create_suggestion(
+            suggestion = suggestion_services.create_suggestion(
```

Comment:
Will it be possible to save the suggestion to datastore only after the image upload process completes? [To avoid creating suggestions whose image isn't available on cloudstorage!]

Reply:
done